### PR TITLE
Update to 1.20.2

### PR DESCRIPTION
**Pull request by MerchantPug**
- Added QuiltMC maven to the repositories block in build.gradle, this was required for their build of GSON, which Origins now uses.
- Added Modrinth maven to the repositories block in build.gradle, now required for Additional Entity Attributes.
- Fixed foodEffectImmunity ModifyVariable inside SuspiciousStewItemMixin, there was an internal change to suspicious stew. Hopefully you don't mind my use of noneMatch, I found streams to be the most efficient here. 
The list was immutable, which is why I went the rebuild route.

I have not messed with the versioning of the mod, you'll have to increment that yourself. :3

### CODE REVIEW

**Comment by electricsteve** [comment]
The build error seems to be a problem with pehkui

**Comment by MerchantPug** [comment]
> The build error seems to be a problem with pehkui

I don't think that there should be a build error, I think that it was just that specific build fucking up. I didn't change anything in the build.gradle between the two commits.

**Comment by electricsteve** [comment]
hmmmm, ok maybe @MoriyaShiine needs to rerun it or just accept pr, my friends want the mod in their 1.20.2 server, but i did look at the file and it indeed did not exist

**Comment by JizzyJack** [comment]
> hmmmm, ok maybe @MoriyaShiine needs to rerun it or just accept pr, my friends want the mod in their 1.20.2 server, but i did look at the file and it indeed did not exist

i have the same problem

**Comment by MerchantPug** [comment]
Confirmed working with Origins 1.12.0.

**Comment by MerchantPug** [comment]
Updated dependencies on the dev side, and slightly bumped FMJ. Didn't bump FMJ all the way to allow it to work without latest loader.

**Comment by JizzyJack** [comment]
> Confirmed working with Origins 1.12.0 without any changes. im a idiot what does that mean (can u dumb it down)

**Comment by JizzyJack** [comment]
> > Confirmed working with origins 1.12.0 without any changes. I just tried it, it isnt working the latest version is 1.20.1
> 
> 1. you didn't quote correctly
> 2. if you are using the version that is on modrinth/curseforge, indeed, the code in the pr is not on modrinth/cf yet.

Is there anyway to show me how to download it with curseforge please, I don’t know how to do it or what your saying

**Comment by electricsteve** [comment]
> > > Confirmed working with origins 1.12.0 without any changes. I just tried it, it isnt working the latest version is 1.20.1
> > 
> > 
> > 
> > 1. you didn't quote correctly
> > 2. if you are using the version that is on modrinth/curseforge, indeed, the code in the pr is not on modrinth/cf yet.
> 
> Is there anyway to show me how to download it with curseforge please, I don’t know how to do it or what your saying

@MoriyaShiine needs to upload it to curseforge first
